src/components/emby-slider/emby-slider.js

```diff
@@ -177,6 +185,116 @@ define(['browser', 'dom', 'layoutManager', 'css!./emby-slider', 'registerElement
         }
     };
 
+    /**
+     * Keyboard dragging timeout.
+     * After this delay "change" event will be fired.
+     */
+    var KeyboardDraggingTimeout = 1000;
```

Comment:
why this one starts with capital `K` while other starts with lower-case `k`?

Reply:
This is const.

Reply:
Not sure we need to differentiate since we cant migrate this codebase to use const or let variables for quite a while.